src/main/webapp/app/config/appConfig.js

```diff
@@ -5,7 +5,7 @@ var appConfig = {
   'allowAnonymous': true,
   'anonymousRole': 'ROLE_ANONYMOUS',
 
-  'authStrategies': ['emailRegistration'],
```

Comment:
Could we please revert the default appConfig.js changes?

Reply:
Reverted